docs/client/admin.rst

```diff
@@ -599,3 +599,73 @@ Optional
 .. cmdoption:: -o, --output
 
     Output the results into a json file
+
+
+-------
+
+Pull a Synapse Project's Wiki
+-----------------------------
+
+Synopsis
+^^^^^^^^
+
+pull-wiki
+    projectid [--workdir]
+
+Description
+^^^^^^^^^^^
+
+Download each wiki page of a synapse project and saves a json configuration
+(wiki_config.json).
+
+Positional
+^^^^^^^^^^
+
+.. program:: challengeutils pull-wiki
+
+.. cmdoption:: projectid
+
+    Synapse project ID e.g. ``syn123455``
+
+Optional
+^^^^^^^^
+
+.. cmdoption:: --workdir
+
+    Path to download markdown files and wiki_config.json
+    Defaults to location of where code is being executed
+
+
+-------
+
+Push a Synapse Project's Wiki
+-----------------------------
+
+Synopsis
+^^^^^^^^
+
+push-wiki
+    projectid [--workdir]
+
+Description
+^^^^^^^^^^^
+
+Updates Synapse wiki by uploading markdown files linked together by a wiki_config.json.
+The markdown and wiki_config.json can be obtained by the ``pull-wiki`` command.
+
+Positional
+^^^^^^^^^^
+
+.. program:: challengeutils push-wiki
+
+.. cmdoption:: projectid
+
+    Synapse project ID e.g. ``syn123455``
+
+Optional
+^^^^^^^^
+
+.. cmdoption:: --workdir
+
+    Path to markdown files and wiki_config.json
+    Defaults to location of where code is being executed
```

Comment:
Is there documentation for `validate-wiki`? Or do we expect a user not use it separate from `push-wiki`?

Regardless, we might want to at least mention that `push-wiki` will validate the config file prior to making any pushes.

Reply:
@vpchung ,  it might clutter up the code less if I just removed the validation cli function since the validation is code is ran anyways when doing a post.  Thoughts?

Reply:
Makes sense to me.